### PR TITLE
[2.1] Update baseline and SDK

### PR DESCRIPTION
**Pull request by JunTaoLuo**
https://github.com/dotnet/aspnetcore-internal/issues/3781

### CODE REVIEW

**Review by dougbu — CHANGES_REQUESTED** [review]

File: build/dependencies.props [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <!-- MicrosoftNETCoreApp21PackageVersion is assigned at the bottom so it can automatically pick up MicrosoftNETCoreAppPackageVersion in an orchestrated build. -->
     <MicrosoftNETCoreAppPackageVersion>2.1.23</MicrosoftNETCoreAppPackageVersion>
     <MicrosoftNETCoreDotNetAppHostPackageVersion>2.1.23</MicrosoftNETCoreDotNetAppHostPackageVersion>
+    <SystemTextEncodingsWebPackageVersion>4.5.0</SystemTextEncodingsWebPackageVersion>
```
[the remark this answers is not in the document]
Comment (JunTaoLuo) [reply build/dependencies.props:7]: You mean repin?

[the remark this answers is not in the document]
Comment (wtgodbe) [reply build/dependencies.props]: And update it to 4.5.1

[the remark this answers is not in the document]
Comment (dougbu) [reply build/dependencies.props]: Yup and yup

**Comment by JunTaoLuo** [comment]
@dougbu did we ever figure out how to fix the timeouts? If not I'm probably going to merge this as is.

**Review by wtgodbe — APPROVED** [review]

**Comment by BrennanConroy** [comment]
https://github.com/dotnet/aspnetcore/pull/30164#issuecomment-791019893

**Comment by dougbu** [comment]
@BrennanConroy's right. I recommend cherry-picking https://github.com/dotnet/aspnetcore/pull/30164/commits/5bafa2d364e6fd3e1bddfca0034b9a81b4e14db7 into this PR. Makes more sense here than in #30164 anyhow.

**Review by dougbu — APPROVED** [review]

**Pull request merged**